.gitignore

```diff
@@ -14,3 +14,7 @@ venv/
 .DS_Store
 .idea
 .vagrant/*
+
+# required for cache slugs
```

Comment:
I think you can add more detail to this comment -- please give enough detail so that someone who is not familiar with all our discussions knows what's going on. It's OK for the comment to span multiple lines.